integration/walletextension/wallet_extension_test.go

```diff
@@ -427,6 +429,25 @@ func TestCannotSubscribeOverHTTP(t *testing.T) {
 	}
 }
 
+func TestSuccessGasEstimate(t *testing.T) {
```

Comment:
@otherview Also, it would be nice if the test name could be aligned with those of all the other tests (e.g. `TestCanEstimateGasAfterSubmittingViewingKey`).

Reply:
Can do 👍